### DISCUSSION TITLE
Tech Twitter Con

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#4225] [Ideas]
We have been joking and bouncing ideas in various places like Twitter. I thought it might be good to capture some ideas in one place where everyone can leave their ideas and thoughts.

Latest ideas are...

| Where | New Delhi |
| :--- | :--- |
| When | ? |
| Ticket Cost | Free? Affordable |
| Sponsors | Yes |
| Duration | 2 days? |
| Size | 200 people? |

Event ideas
- Talks (standard)
- Workshops (standard)
- Panel discussions
- Speed networking
- Ice breaker games (we need the fun stuff too)

note: this is all subject to change

**Comment by eddiejaoude** [comment]
Leave a message below if you want to be a speaker 👇

**Reply by FrancescoXX** [reply]
I can be a speaker for:
- Motivational (Introverted Developers on Social Media)
- Web3: Intro to blockchain and Solidity

I can also make a workshop for Docker 🐳

**Reply by eddiejaoude** [reply]
I can speak on:

- Open Source (no surprise there 😂 )
- GitHub (no surprise there either 😂 )
- Freelancing
- Personal branding
(I will think of a few more too)

I think we should some panels too with Q&A from the audience, like a TwitterSpace but in real life! 🚀

**Reply by InHuOfficial** [reply]
I can speak on:
- accessibility
- web vitals

Probably some other things too, but I can fill any length slot on those topics!

**Reply by suitupstan** [reply]
I can speak on:

- networking
- open source
- web3 theory
- community management 
- communities 🎉 

I can also host mini events n all

**Reply by khardikk** [reply]
I can speak on :

- no/low code development
- community management

**Reply by bhavukkalra** [reply]
I can speak on:

- Cloud Computing
- Community 
- Student and Professional Programs
- Research Programs
- and Of course (Open Source)

**Reply by Pradumnasaraf** [reply]
I can speak on:

- Open Source 
   - Community standards.
   - Best practice as a maintainer.
   - Best practice as a contributor.
   
- Build in public and ask in public movement.

**Reply by SuperAayush** [reply]
I can speak on:

- networking
- open source
- remote work
- communities 
- development

**Reply by techwithbhavya** [reply]
I can speak on :
<li>
Networking
</li>
<li>
Open-source
</li>
<li>
Internships
</li>
<li>
Remote work
</li>
<li> Communities </li>
<li> Development </li>
<li> Ambassadors program and how it will improve your soft skills ? </li>

**Reply by indevi** [reply]
I can speak on:
- Non code contributions
- Getting started with open source 
- LFX and other remote externships

**Reply by mfts** [reply]
I can speak on:

- open source
- hackathons
- developer leadership
- building video conferencing from scratch for 1M users/month 
- how to raise from VC as tech founder
- CTO turned VC

**Reply by spences10** [reply]
I can speak on:
- svelte
- blogging
- jamstack

I can do workshops on using Svelte with GraphQL and SvelteKit

**Reply by ykdojo** [reply]
I can speak on:
 
- tech twitter growth tips
- twitter spaces
- potentially moderate a few panels - like twitter spaces IRL as Eddie mentioned

**Reply by rebelchris** [reply]
I can speak on:

- fast web
- consistency
- burnouts
- astro
- monorepos

**Reply by lovelacecoding** [reply]
I can speak on:
* Social media growth
* Study / certification prep
* .NET / C# Development
* Python Development
* Web Accessibility

**Reply by SwapnilMane141** [reply]
I can speak on:

- Freelancing
- Web Development
- Data Extraction / Web scraping
- Python
- Networking

**Reply by deleteman** [reply]
> Leave a message below if you want to be a speaker 👇

Heck yeah, how did I miss this?!

I can speak about:
- getting started as a dev
- writing and content creation for devs aspiring to become  tech writers 
- JavaScript
- Big data and software architecture
- Front-end observability, debugging your front-end using your user's computer
- The importance of content for open source companies
And probably something else I can't think of right now.

**Reply by sumusiriwardana** [reply]
I can speak on:

- Leadership
- Team Management
- Networking
- Content creations (As introverts)
- Async team culture
- Personal productivity 
- Agile/Scrum/Kanban

**Reply by vfarcic** [reply]
A fireside chat or AMA session with a few other speakers.

**Reply by Ak-Gautam** [reply]
I have conducted workshops previously on and can speak on:
- Robotics (more hardware oriented)
- Data Science
- Machine Learning
- Student internships
- IoT (more hardware oriented)

**Reply by rohitg00** [reply]
I can speak on:

-  DevOps
- My DevOps Journey
- Getting into DevOps by learn by doing and public sharing
- What is MLOps, Why to learn this tech? 
- Ace the Remote Job interviews
- How open-source help in getting remote jobs

**Reply by vrajdesai78** [reply]
I can speak on:

1. How open source helps you to take to places?
2. How to network as a student and get out of imposter syndrome. 
3. How to get started with web3 and the future of blockchain.
4. Future of Mobile Development

**Reply by vanshhhhh** [reply]
I can speak on:

* Open Source and Open Source Programs (Google Summer of Code particularly as I am one of the contributor this year)
* Machine Learning and Deep Leaning
* Research Programs

**Comment by eddiejaoude** [comment]
Leave a message below if you want to attend the event 👇

**Reply by InHuOfficial** [reply]
Interested in principle! Depends on when!

**Reply by patilharsh03** [reply]
I'm interested in attending the event. 

Because it's gonna be fun and I can meet a lot of great people and developers in tech and learn something new from them.

**Reply by octonawish-akcodes** [reply]
I want to attend but it depends where and when its going to happen

**Reply by vaibhav0726** [reply]
Yes.
And here are some reasons:-
 1. I want to listen you(@eddiejaoude)  and @FrancescoXX  speeches.
 2. Wants gain knowledge by meeting tech twitter people.

**Reply by khardikk** [reply]
Interested to attend

**Reply by Sreekar-Shodhan** [reply]
I would love to attend the event in person but I have few health issues so I am not allowed to travel or eat any kind of outside food. If possible can you please stream online . Atleast one event like maybe 2 hrs at least.

**Reply by zelfroster** [reply]
I definitely want to attend the event as It will be my first event and literally can't wait to meet the amazing people in real life ☺️.

**Reply by suitupstan** [reply]
I would love to, but timings matter

**Reply by bhavukkalra** [reply]
I would love to attend this In person event❤

**Reply by harshsinghatz** [reply]
Always in, to meet some great folks!!

**Reply by Pradumnasaraf** [reply]
Yes for sure :)

**Reply by Santosh1176** [reply]
Surely! I would love to attend.

**Reply by karnatisrinivas** [reply]
Surely, would love to ❤

**Reply by VaibhavSingh8** [reply]
Definitely, I would love to attend the event. ;)

**Reply by anuragh2002** [reply]
I'll attend the event for sure, count me in.

**Reply by prabhpreet-code** [reply]
Aahhh... Very excited to join you all in-person !!! It's one of the best opportunities to learn from the best folks out there and gain invaluable insights.🔥💯

**Reply by SuperAayush** [reply]
I want to attend for sure!!

**Reply by techwithbhavya** [reply]
I want to attend for sure. Count me in. It is one of the best opportunities to learn and interact with the amazing folks out there.

**Reply by indevi** [reply]
Thrilled to be a part of this event✨

**Reply by spences10** [reply]
Yep, sure, for fee too 🥰

**Reply by juliafmorgado** [reply]
I'm def in!

**Reply by suhailkakar** [reply]
Yes, count me in :) 💯

**Reply by ykdojo** [reply]
As I mentioned, it'd be super cool if it's in-person and live-streamed at the same time

**Reply by Honey-9** [reply]
I'm Interested Finally we can have Tech Twitter Con

**Reply by rebelchris** [reply]
Depends on time and place :)

**Reply by sajalshlan** [reply]
Very much interested

**Reply by lovelacecoding** [reply]
Really depends on the time and place, university and all that stuff, BUT I would absolutely love to.

**Reply by SwapnilMane141** [reply]
Yes... Would love to attend. Can't wait to geek out!

**Reply by conermurphy** [reply]
Would 100% be interested in attending, depending on location and timing though. 😃

**Reply by kitarp29** [reply]
I wanna attend!!!
![giphy](https://user-images.githubusercontent.com/56778773/177054650-c8844f02-5d71-4f17-9c99-1a2a6e1dc852.gif)

**Reply by sarkardeep** [reply]
I would love to attend , It's gonna be super amazing

**Reply by 889-dj** [reply]
Wanted to attend as I m eager to meet all the great folks from the community also I love to network and meet new folks and learn from the community 🙂🤩🤩

**Reply by akrsh24** [reply]
I am definitely in.

**Reply by deleteman** [reply]
Love to attend, I just need to know when and where/how

**Reply by sumitsaurabh927** [reply]
Depends on timings and availability but I'd love to be with y'all!

**Reply by sumusiriwardana** [reply]
I want to attend!

**Reply by grbz378** [reply]
would love to know more before final confirmation!

**Reply by Ak-Gautam** [reply]
I am a student and I would most definitely like to attend!

**Reply by Prahitha** [reply]
Would love to attend!

**Reply by harshavgaddam** [reply]
I would like to attend based on Location.

**Reply by himanshubalani** [reply]
Oh I'd love to!!

**Reply by asdsyd** [reply]
I am interested!

**Reply by RishabhDhawad** [reply]
Yes definitely I want to attend this event!

**Reply by Mubashir-Md** [reply]
Would love to attend 😍... And yeah meeting Kunal, Eddie, Francesco IRL and many other tech folks would be awesome 😄...

**Reply by thealeemquadri** [reply]
Hey, I will be glad to attend and meet you.

**Reply by rohitg00** [reply]
I will be glad to attend this event.

**Reply by asdsyd** [reply]
Please try to keep it in first week of november we also have MLH event partner hackathon on 6-7 Nov in Delhi so it overlaps. It would be very helpful for people travelling to delhi. Rest upto you eddie. :)

**Comment by eddiejaoude** [comment]
Leave a message below if you want to be in @eddiejaoude's YouTube vlog 👇

**Reply by FrancescoXX** [reply]
I def want!

**Reply by InHuOfficial** [reply]
General Vlog or a vlog about this event? It is yes to both, just wanted clarification lol!

**Reply by octonawish-akcodes** [reply]
I also want to be in it

**Reply by patilharsh03** [reply]
I want to be in that vlog

**Reply by vaibhav0726** [reply]
Yes definitely. It seems like dream will becomes true.

**Reply by suitupstan** [reply]
def, is that a question?

**Reply by zelfroster** [reply]
Would love to be in your vlog :)

**Reply by bhavukkalra** [reply]
Who wouldn't want to be in? Of course, yes

**Reply by harshsinghatz** [reply]
Yep!!

**Reply by Pradumnasaraf** [reply]
I am in!

**Reply by karnatisrinivas** [reply]
Is that even a question??

**Reply by VaibhavSingh8** [reply]
YES SIRRRR! I am IN.

**Reply by anuragh2002** [reply]
You can't miss that, so I'm def in.

**Reply by SuperAayush** [reply]
Yes I want to be in those famous Vlogs

**Reply by techwithbhavya** [reply]
Yes ofc, I want to be in those amazing Vlogs.

**Reply by indevi** [reply]
+1 with that

**Reply by spences10** [reply]
Yes, deffo

**Reply by juliafmorgado** [reply]
Always!

**Reply by suhailkakar** [reply]
Why not :)
Differently!!

**Reply by Honey-9** [reply]
For Sureee

**Reply by sajalshlan** [reply]
I definitely want in!

**Reply by SwapnilMane141** [reply]
Yes, Offcourse Eddie!

**Reply by 889-dj** [reply]
Definitely it's a big yessss😌

**Reply by deleteman** [reply]
Love to, not sure what it implies, but count me in!

**Reply by sumitsaurabh927** [reply]
Absolutely! 🤘

**Reply by sumusiriwardana** [reply]
A big yes!

**Reply by rohitg00** [reply]
Wow, I don't wanna miss that iconic pose from eddie.

**Comment by eddiejaoude** [comment]
Leave event ideas below (for example: speed networking, open source corner, games, ...etc) 👇

**Reply by FrancescoXX** [reply]
having a fireside chat with you Eddie!

**Reply by eddiejaoude** [reply]
We should do some panels with Q&As, it will be like a IRL TwitterSpace 😍

**Reply by FrancescoXX** [reply]
ahaahha let's calle them like that? IRL Twitter Spaces 😂

**Reply by InHuOfficial** [reply]
So to move my ideas (very rough) from DMs to here (I think a couple mirror what has been said already 🤣):

Instead of this being a conference, think of it more like a networking event.

That way the event can be positioned as "team building and talent acquisition" to potential sponsors. They can have booths with games etc. to promote their products etc.

Instead of singular speakers on a subject there are small panels on a subject and we run IRL "spaces". With a bit of trickery we could potentially broadcast these "spaces" via live stream (YT, Twitch etc.) and even audio on a Twitter space and make them interactive for people who are not at the event in person (but preference for people who are there obviously)

Also run "speed dating" type sessions where you get 2-3 minutes to sit with a random dev and then move on. This allows loads of people to meet potential new friends!

**Reply by suitupstan** [reply]
Rant session 😂

**Reply by bhavukkalra** [reply]
Separate discussion rooms with specific topics. Through this we won't be restricting anyone to a particular topic, They can explore

**Reply by Pradumnasaraf** [reply]
Just a random idea "Opening a PR session". In that attendees can their names and pictures to the repo, which is dedicated to this conf. It will kind of collective and memorable thing. 100s of PR, kinda a record-making thing :)

**Reply by karnatisrinivas** [reply]
> Rant session 😂

It is cool 😂

**Reply by prabhpreet-code** [reply]
> 

Amazing idea indeed!!! It's a special one for the folks to get started with open source...🚀

**Reply by SuperAayush** [reply]
Can we have some small competitions like:

- Review a set of codes and you might win some vouchers or stuff (we can teach alongside by taking good examples).
- Pitch your Open-Source projects and if it has the best set of practices followed again some swags or vouchers(we can ask few projects in the google form pre-event and select a few of them for reviewing offline 🤔)[again we can teach alongside with some good examples].
- Since it TwitterCon we can have a fun event by sharing the Twitter Header of some folks with good following and asking participants to guess the person.
- I think these events miss dancing and singing(ik it is bit unprofessional but I think music and dance brings even more fun to the event).

These are my ideas I hope you like them 😄.

**Reply by mfts** [reply]
I’m all for panels. 

I like mini-biographies presentations (10mins) from people in tech across different stages (junior dev…CTO, engineering manager etc)

**Reply by techwithbhavya** [reply]
> Can we have some small competitions like:
> 
> * Review a set of codes and you might win some vouchers or stuff (we can teach alongside by taking good examples).
> * Pitch your Open-Source projects and if it has the best set of practices followed again some swags or vouchers(we can ask few projects in the google form pre-event and select a few of them for reviewing offline 🤔)[again we can teach alongside with some good examples].
> * Since it TwitterCon we can have a fun event by sharing the Twitter Header of some folks with good following and asking participants to guess the person.
> * I think these events miss dancing and singing(ik it is bit unprofessional but I think music and dance brings even more fun to the event).
> 
> These are my ideas I hope you like them 😄.

Yup +1 to this. I really like this.

**Reply by 889-dj** [reply]
We can have a open source project ideation pool sort of event where folks can share their ideas of a product that they wanna create or already it's in the production and the community can help reach them their true potential and accomplish their goals 🙂

**Reply by sumusiriwardana** [reply]
> So to move my ideas (very rough) from DMs to here (I think a couple mirror what has been said already 🤣):
> 
> Instead of this being a conference, think of it more like a networking event.
> 
> That way the event can be positioned as "team building and talent acquisition" to potential sponsors. They can have booths with games etc. to promote their products etc.
> 
> Instead of singular speakers on a subject there are small panels on a subject and we run IRL "spaces". With a bit of trickery we could potentially broadcast these "spaces" via live stream (YT, Twitch etc.) and even audio on a Twitter space and make them interactive for people who are not at the event in person (but preference for people who are there obviously)
> 
> Also run "speed dating" type sessions where you get 2-3 minutes to sit with a random dev and then move on. This allows loads of people to meet potential new friends!

Love this idea!

**Reply by rohitg00** [reply]
Games and brainstorming ideas

**Comment by eddiejaoude** [comment]
Q. Who do you want to meet? Let us know below 👇

**Reply by InHuOfficial** [reply]
Who do you want to me**et** I think this should be!

**Reply by eddiejaoude** [reply]
Thank you, updated 👍

**Reply by vaibhav0726** [reply]
If got a chance to attend then definitely yes.

**Reply by suitupstan** [reply]
Everyone, specially @eddiejaoude

**Reply by zelfroster** [reply]
Well everyone if possible.
And I have got a long list of people to meet ❤️.

**Reply by bhavukkalra** [reply]
Everyone!

**Reply by karnatisrinivas** [reply]
Everyone

**Reply by VaibhavSingh8** [reply]
Everyone who I follow on tech twitter, I would love to meet them!

**Reply by SuperAayush** [reply]
There are so many folks I want to meet, If I start tagging them that will be a long list 😄

**Reply by techwithbhavya** [reply]
There are so many amazing folks I really want to meet. it will be a long list if I start tagging all the folks out there.😀

**Reply by indevi** [reply]
🎉Depends on the attendees, but i am particularly excited to meet Aditya Oberoi

**Reply by ykdojo** [reply]
yes

**Reply by SwapnilMane141** [reply]
Everyone... So many amazing people on the list!

**Reply by sarkardeep** [reply]
I'm Planning to meet everyone !

**Reply by 889-dj** [reply]
I wanna meet all the folks from the community also I would love to discuss about the content creation and related stuff with u @eddiejaoude  and @FrancescoXX 🤩

**Reply by sumitsaurabh927** [reply]
Everyone!

**Reply by grbz378** [reply]
@eddiejaoude @FrancescoXX 
@emmalearnscode @Pradumnasaraf 
@sumusiriwardana @InHuOfficial 

and many more...would love to make new friends if I could attend!

**Comment by eddiejaoude** [comment]
Q. Name ideas for the event? 👇 

@FrancescoXX suggested `TTCon` but I think we can do better 😜

**Reply by patilharsh03** [reply]
Tech Twitter fest

**Reply by suitupstan** [reply]
TechCon -> tecon

**Reply by zelfroster** [reply]
TwiCon

**Reply by bhavukkalra** [reply]
TweepCon or TweeCon

**Reply by SuperAayush** [reply]
twtcon

**Reply by spences10** [reply]
Tech Twitter IRL

**Reply by suhailkakar** [reply]
+1 @spences10

**Reply by InHuOfficial** [reply]
How about Twirl Con then? (**T**ech t**W**itter **I**n **R**eal **L**ife **Con**ference!)

**Reply by arbindmishra** [reply]
TwechFerence

**Reply by sumitsaurabh927** [reply]
TechniCron....as in TECHNIcal CReator's cONference

**Reply by grbz378** [reply]
TechniCon

**Reply by rohitg00** [reply]
Twt kaun?

**Comment by eddiejaoude** [comment]
Q. Do you want to get involved and be a helper before or during the event?

**Reply by octonawish-akcodes** [reply]
Yes, I can help

**Reply by khardikk** [reply]
Yes

**Reply by suitupstan** [reply]
I would love to be (I live in Delhi too btw 😁)

**Reply by zelfroster** [reply]
I can help too.

**Reply by harshsinghatz** [reply]
I would love to help!!

**Reply by Pradumnasaraf** [reply]
Always open to help!!

**Reply by karnatisrinivas** [reply]
Always open to help

**Reply by VaibhavSingh8** [reply]
I will be glad to help.
(not sure for before event, depends on availability). During event, For sure!

**Reply by prabhpreet-code** [reply]
Yeahh!! It would be a pleasure to contribute...

**Reply by SuperAayush** [reply]
Would love to help

**Reply by techwithbhavya** [reply]
I will be happy to help.

**Reply by indevi** [reply]
will be very excited to

**Reply by spences10** [reply]
I can help emcee

**Reply by juliafmorgado** [reply]
YES!

**Reply by Honey-9** [reply]
Yess!!

**Reply by InHuOfficial** [reply]
If you need me just ask, looks like you have plenty of volunteers but obviously I will throw down if needed!

**Reply by sarkardeep** [reply]
Yes , I would love to , let me know if I can help

**Reply by 889-dj** [reply]
Will love to help !! 😀

**Reply by akrsh24** [reply]
It will be a pleasure to help in this. Count me in.

**Reply by Prahitha** [reply]
Happy to help!

**Reply by meghanacosmos** [reply]
Sure, would love to!

**Reply by himanshubalani** [reply]
Yes!

**Reply by rohitg00** [reply]
I would love to be, But I don't stay in delhi

**Comment by bhavukkalra** [comment]
This is subject to change as I don't live in delhi. But If timings as favorable then I can come to delhi early as well and assist

**Reply by InHuOfficial** [reply]
I think you meant to reply not raise a new discussion item? 😜🤣

**Comment by Ananya2001-an** [comment]
Is this still happening? I saw that this discussion started in last year. TBH I would love to attend this event if it happens in the future 👍🏼

**Reply by Pradumnasaraf** [reply]
Unfortunately, it's not possible at this time. While we would all love to attend and make this event happen someday, it's not feasible in the near future. Thank you for asking. Furthermore, I will be closing this conversation. 

**Thank you to everyone who participated and showed interest.**
